PREINSTALL.md

```diff
@@ -0,0 +1,42 @@
+Use this extension to synchronize documents from a Cloud Firestore collection to a Meilisearch index. This allows you to use a full text search in your Cloud Firestore documents.
+
+This extension listens to each creation, update or deletion of your documents to keep them sync with your Meilisearch index, so the data in Meilisearch is a mirror of your content in Cloud Firestore. You can then run queries on this mirrored dataset.
+
+Note that this extension only listens for changes to _documents_ in a specific collection, but not changes in any _subcollection_. However, you can install additional instances of this extension to listen to other collections in your Firestore database.
+
+#### Additional setup
+
+Before installing this extension, you'll need to:
+
+- [Set up Cloud Firestore in your Firebase project.](https://firebase.google.com/docs/firestore/quickstart)
+- Run Meilisearch instance. There are many easy ways [to download and run a Meilisearch instance.](https://docs.meilisearch.com/learn/getting_started/installation.html#download-and-launch)
+
+#### Data import format
+
+Documents indexed in Meilisearch need to have a [unique id](https://docs.meilisearch.com/learn/core_concepts/documents.html#primary-field), to do so this extension will use the default field provided for this purpose in Firestore: `Document ID` which will be renamed `_firestore_id` used as [document id](https://docs.meilisearch.com/learn/core_concepts/documents.html#document-id). If your documents have an `id` field it will be added to your documents without it being your primary key.
```

Comment:
> to do so this extension will use the default field provided for this purpose in Firestore: `Document ID` which will be renamed `_firestore_id` used as [document id]

Not sure I understand this sentence

Reply:
I rewrite it not sure it better let me know